packages/blob-reader/helper.js

```diff
@@ -63,40 +61,47 @@ function getPath(el) {
     );
   }
 
-  let ret = $(rootSelector)
-    .filter(function () {
-      return $(this).text().trim() === 'View file';
-    })
-    .attr('href');
+  const [link] = Array.from(rootSelector).filter((value) => {
+    return value.textContent.trim() === 'View file';
+  });
+
+  let ret = link && link.getAttribute('href');
 
   if (!ret) {
-    ret = $('.js-permalink-shortcut').attr('href');
+    const shortcut = document.querySelector('.js-permalink-shortcut');
+
+    ret = shortcut && shortcut.getAttribute('href');
```

Comment:
Again, I think optional chaining helps to make this more readable.

```suggestion
    ret = document.querySelector('.js-permalink-shortcut')?.getAttribute('href');
```